src/vs/workbench/services/extensions/common/extensionsRegistry.ts

```diff
@@ -393,6 +393,11 @@ export const schema: IJSONSchema = {
 						body: 'onChatParticipant:${1:participantId}',
 						description: nls.localize('vscode.extension.activationEvents.onChatParticipant', 'An activation event emitted when the specified chat participant is invoked.'),
 					},
+					{
+						label: 'onLanguageModelChat',
+						body: 'onLanguageModelChat:${1:vendor}',
+						description: nls.localize('vscode.extension.activationEvents.onLanguageModelChat', 'An activation event emitted when a chat model provider for the given vendor is requested.'),
+					},
```

Comment:
Is this a new activation event? They need to go through the API call.

Reply:
This is an existing activation event, that is just havine intellisense added to it. You can see the activation event has existed awhile in [main](https://github.com/microsoft/vscode/blob/7e6267da4ea06c526b2e2d249db3448b51229e15/src/vs/workbench/contrib/chat/common/languageModels.ts#L286-L291) and has already been discussed in the API call